actions/views/rhs.js

```diff
@@ -141,48 +141,20 @@ function getPostRHSSearchActions(searchPostSuccess, result, teamId) {
     return [...searchActions, {type: searchPostSuccess}];
 }
 
-export function getFlaggedPosts() {
-    return async (dispatch, getState) => {
-        const state = getState();
-        const userId = getCurrentUserId(state);
-        const teamId = getCurrentTeamId(state);
-
-        const result = await Client4.getFlaggedPosts(userId, '', teamId);
-
-        await PostActions.getProfilesAndStatusesForPosts(result.posts, dispatch, getState);
-
-        const searchActions = getSearchActions(result, teamId);
-
-        dispatch(batchActions(searchActions));
-    };
-}
-
 export function showFlaggedPosts() {
     return async (dispatch, getState) => {
         const state = getState();
-        const userId = getCurrentUserId(state);
         const teamId = getCurrentTeamId(state);
 
-        const preRHSSearchActions = getPreRHSSearchActions(
-            ActionTypes.SEARCH_FLAGGED_POSTS_REQUEST,
-            '',
-            RHSStates.FLAG
-        );
-
-        dispatch(batchActions(preRHSSearchActions));
-
-        let result;
-        try {
-            result = await Client4.getFlaggedPosts(userId, '', teamId);
-        } catch (error) {
-            dispatch({type: ActionTypes.SEARCH_FLAGGED_POSTS_FAILURE, error});
-        }
+        dispatch({
```

Comment:
At first, I was thinking we should rup out handling of `ActionTypes.SEARCH_FLAGGED_POSTS_REQUEST` everywhere too, until I realized that mattermost-redux uses an action type of the same name here, so everything "just works", though "scary" to see the action name overlap.

In the future, it would be interesting to see if there is a parallel change for pinned posts to eliminate the `get(PrePost)RHSSearchActions` methods altogether.